tutorials/SDF/car_demo.md

```diff
@@ -0,0 +1,520 @@
+                <property type="double" key="z">1</property>
+
+                <property type="string" key="state">floating</property>
+                <anchors target="3D View">
+                    <line own="right" target="right"/>
+                    <line own="bottom" target="bottom"/>
+                </anchors>
+                </ignition-gui>
+
+                <sim_time>true</sim_time>
+                <real_time>true</real_time>
+                <real_time_factor>true</real_time_factor>
+                <iterations>true</iterations>
+                <topic>/world/car_world/stats</topic>
+
+            </plugin>
+
+            <!-- Entity tree -->
+            <plugin filename="EntityTree" name="Entity tree">
+            </plugin>
+
+        </gui>
+
+        <light type="directional" name="sun">
+            <cast_shadows>true</cast_shadows>
+            <pose>0 0 10 0 0 0</pose>
+            <diffuse>0.8 0.8 0.8 1</diffuse>
+            <specular>0.2 0.2 0.2 1</specular>
+            <attenuation>
+                <range>1000</range>
+                <constant>0.9</constant>
+                <linear>0.01</linear>
+                <quadratic>0.001</quadratic>
+            </attenuation>
+            <direction>-0.5 0.1 -0.9</direction>
+        </light>
+
+        <model name="ground_plane">
+            <static>true</static>
+            <link name="link">
+                <collision name="collision">
+                <geometry>
+                    <plane>
+                    <normal>0 0 1</normal>
+                    </plane>
+                </geometry>
+                </collision>
+                <visual name="visual">
+                <geometry>
+                    <plane>
+                    <normal>0 0 1</normal>
+                    <size>100 100</size>
+                    </plane>
+                </geometry>
+                <material>
+                    <ambient>0.8 0.8 0.8 1</ambient>
+                    <diffuse>0.8 0.8 0.8 1</diffuse>
+                    <specular>0.8 0.8 0.8 1</specular>
+                </material>
+                </visual>
+            </link>
+        </model>
+    </world>
+</sdf>
+```
+
+Save the file and try to launch it.
+
+`ign gazebo car_demo.sdf`
+
+**Note**: you can name your file any name.
+
+You should see an empty world with just a ground plane. Check [World demo](../world_demo.md) to learn how to build your own world.
+
+## Building a model
+
+Under the `</model>` tag we will add our robot model as follows:
+
+### Defining the model
+
+```xml
+<model name='vehicle_blue' canonical_link='chassis'>
+    <pose relative_to=world>0 0 0 0 0 0</pose>
+```
+
+Here we define the name of our model `vehicle_blue`, which should be a unique name among its siblings (other tags or models on the same level).
+Each non-static model must have at least one link designated as the `canonical_link`; the implicit frame of the model is attached to this link.
```

Comment:
```suggestion
Each model may have one link designated as the `canonical_link`; the implicit frame of the model is attached to this link. If not defined, the first `<link>` will be chosen as the canonical link.
```

The [canonical_link](http://sdformat.org/spec?ver=1.7&elem=model#model_canonical_link) tag is optional. I also think it applies to static models as well.